### DISCUSSION TITLE
Pre-cracked sample

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#306] [Q&A] [answered]
Hello, 

I have modeled one rectangular plate with an existing crack in its center, I want to see how the crack propagates, but that is not happening in my simulation. I am attaching an image of my sample and the YAML file below; please tell me what I am doing wrong.

![Screenshot 2023-03-13 134320](https://user-images.githubusercontent.com/110670051/224644409-35d92aa3-216a-40f7-b67d-01d7130fde69.png)

Peridigm:
  Verbose: false
  Discretization:
    Type: "Exodus"
    Input Mesh File: "Plate.g"
  Materials:
    My Material:
      Material Model: "Elastic"
      Density: 8.0
      Bulk Modulus: 1.500e12
      Shear Modulus: 6.923e11
  Blocks:
    My Block:
      Block Names: "block_1"
      Material: "My Material"
      Horizon: 0.8
  Damage Models:
          Damage Model: "Critical Stretch"
          Critical Stretch: 0.0005
  Boundary Conditions:
      Initial Velocity X:
          Type: "Initial Velocity"
          Node Set: "nodelist_1"
          Coordinate: "x"
          Value: "5.08"
      Initial Velocity X2:
          Type: "Initial Velocity"
          Node Set: "nodelist_2"
          Coordinate: "x"
          Value: "-5.08"
      Initial Velocity Y:
          Type: "Initial Velocity"
          Node Set: "nodelist_3"
          Coordinate: "y"
          Value: "0"
      Initial Velocity Z:
          Type: "Initial Velocity"
          Node Set: "nodelist_4"
          Coordinate: "z"
          Value: "0"
   Solver:
    Initial Time: 0.0
    Final Time: 0.0008
    Verlet:
      Safety Factor: 0.7
  Output:
    Output File Type: "ExodusII"
    Output Format: "BINARY"
    Output Filename: "PrecrackedPlate"
    Output Frequency: 1
    Parallel Write: true
    Output Variables:
      Displacement: true
      Velocity: true
      Element_Id: true
      Proc_Num: true
      Dilatation: true
      Force_Density: true
      Force: true
      Volume: true
      Damage: true

**Comment by oldninja** [comment]
This will not work because those elements are too small near the crack and are probably being accounted for as neighbors in your horizon. I would suggest to have the same rectangle but without the cutout in the mesh and use a bond filter to remove those bonds in the input file. See one of the tests cases.

**Comment by Nikhil23121998** [comment]
![image](https://user-images.githubusercontent.com/110670051/225064962-65bbd489-8ec8-4e48-9fc3-bb11319ecc57.png)

This is how the final result is showing after applying bond filter and using Delaunay3d filter in ParaView. Which filter can I use for seeing the propagation of cracks?

**Comment by djlittl** [comment]
You can just set the color to display the "Damage" variable.  Looks like it's currently set to velocity, or similar.  Setting the color is one of the pull-down menus in ParaView, you don't need to use a filter.

**Reply by Nikhil23121998** [reply]
Sir, this is how it looks in the damage variable. 

![Screenshot (68)](https://user-images.githubusercontent.com/110670051/225240199-7d59e59b-ca1b-4199-945d-f9a2e781c5cb.png)

**Comment by djlittl** [comment, marked as answer]
Well, it certainly looks like there is no damage, and hence no crack propagation.  The most likely cause is that the loading is insufficient to cause the material to fail.  I notice that you're using an initial condition on the velocity.  Perhaps you should try a prescribed displacement on the ends of the plate, and pull it apart such that there is no doubt that the specimen should fail (e.g., pull it to 10% strain).  It's possible that the initial velocity is just causing waves to bounce around in the plate without pulling it apart sufficiently to cause it to fail.

**Reply by Nikhil23121998** [reply]
Ok, thank you sir
